components/gcp/automl/create_dataset_for_tables/component.py

```diff
@@ -64,5 +64,6 @@ def automl_create_dataset_for_tables(
         packages_to_install=['google-cloud-automl==0.4.0'],
         annotations={
             "author": "Alexey Volkov <alexey.volkov@ark-kun.com>",
+            "canonical_location": "https://raw.githubusercontent.com/Ark-kun/pipeline_components/master/components/gcp/automl/create_dataset_for_tables/component.yaml",
```

Comment:
Can you explain why are canonical locations needed? Why is the canonical location of GCP components in your private repo? 

/cc @IronPan